.rubocop.yml

```diff
@@ -131,4 +131,4 @@ Lint/SelfAssignment:
   Enabled: false
 Lint/RedundantCopDisableDirective:
   Enabled: false
-
+  
```

Comment:
cant work out how to get rid of this but i think more faff than its worth to remove from the commit now lol